.dockerignore

```diff
@@ -1,9 +1,16 @@
-.vagrant/
-_build/
-apps/aecuckoo/c_src
-apps/aecuckoo/priv/bin
-apps/aecuckoo/priv/lib
-apps/aecore/priv/keys
-system_test
-erl_crash.dump
-aevm_external
+*
+!rebar.config
+!rebar.lock
+!Makefile
+!rebar3
+!rebar.config.script
+!VERSION
+!apps
+!.git
```

Comment:
Why? I suspect this defeats whitelist.

And - *if* this needed - .gitignore to be considered.

Reply:
I guess the only reason `https://github.com/aeternity/epoch/blob/master/rebar.config#L161`

Reply:
yes that's correct, it is failing build without .git